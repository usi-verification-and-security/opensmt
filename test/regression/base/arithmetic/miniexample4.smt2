(set-logic QF_LRA)
(declare-fun x () Real)
(declare-fun y () Real)
(define-fun uninterp_div ((a Real) (b Real)) Real (/ a b))
(assert (= (uninterp_div y x) 20))
(check-sat)
(get-model)
(exit)
