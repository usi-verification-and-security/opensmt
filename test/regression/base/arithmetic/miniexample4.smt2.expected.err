terminate called after throwing an instance of 'opensmt::LANonLinearException'
  what():  Term (* -1 (/ y x)) is non-linear
Aborted
