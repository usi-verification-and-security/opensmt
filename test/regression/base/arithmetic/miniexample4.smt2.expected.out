Term (/ y x) is non-linear
unknown
